src/workers/cdc.ts

```diff
@@ -108,35 +108,31 @@ const onSourceRequestChange = async (
 ): Promise<void> => {
   if (data.payload.op === 'c') {
     // New source request
-    await notifySourceRequest(
-      logger,
-      NotificationReason.New,
-      data.payload.after,
-    );
+    await triggerTypedEvent(logger, 'pub-request', {
```

Comment:
Here you get autocomplete when typing the topic and also when setting the object + type validation